src/node_process.cc

```diff
@@ -587,6 +587,11 @@ void InitGroups(const FunctionCallbackInfo<Value>& args) {
 
 #endif  // __POSIX__ && !defined(__ANDROID__) && !defined(__CloudABI__)
 
+void GetProcessId(v8::Local<v8::Name> property,
+                  const v8::PropertyCallbackInfo<v8::Value>& info) {
+  info.GetReturnValue().Set(Integer::New(info.GetIsolate(), uv_os_getpid()));
```

Comment:
No need to create an Integer, there are Set() overloads that accept the return value of `uv_os_getpid()` directly.